#!/usr/bin/env python
PACKAGE = "multisense_ros"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

#gen.add("auto_exposure", bool_t, 0, "AutoExposure", True)
gen.add("exposure_time", double_t, 0, "Exposure", 0.025, 0, 0.1)

gen.add("auto_gain", bool_t, 0, "AutoGain", True)
gen.add("gain", double_t, 0, "Gain", 1.0, 0, 10)

gen.add("fps", double_t, 0, "FPS", 10, 0, 30)

exit(gen.generate(PACKAGE, "Camera", "Camera"))
